src/services/api.ts

```diff
@@ -78,7 +78,8 @@ export class YdbEmbeddedAPI extends AxiosWrapper {
             // authUrl - external auth service link, after successful auth additional cookies will be appended
             // that will allow access to clusters where OIDC proxy is a balancer
             if (response && response.status === 401 && response.data?.authUrl) {
-                return window.location.assign(response.data.authUrl);
+                window.location.assign(response.data.authUrl);
+                return Promise.reject(error);
```

Comment:
`window.location.assign` doesn't happen immediately, return error so some normal error will be shown in between